network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -36,25 +138,19 @@ class ProductPage(FoundationMetadataPageMixin, Page):
     )
     review_date = models.DateField(
         help_text='Review date of this product',
+        null=True,
```

Comment:
this should only be `null=True` inside a migration, I assume follow-up PRs give this a meaningful value and then remove it again?

Reply:
You're right, this should mimic what we did with the `review_date` Product model. I'll remove this and update the migration file. Good catch!